keydb/templates/secret-utils.yaml

```diff
@@ -46,4 +46,10 @@ stringData:
         {{- end }}
         {{- end }}
         {{- end }}
-        $(echo "${replicas}" | xargs)
+        $(echo "${replicas}" | xargs))
+    {{- if .Values.existingSecret }}
```

Comment:
This isn't enough as there are also `.Values.password` affecting to `REDIS_PASSWORD`.  Just check whether `REDIS_PASSWORD` is non-empty.

Reply:
Fixed based on this answer: https://stackoverflow.com/a/13864829